### ISSUE TITLE
ComputingElement: Path not found warning

### ISSUE THREAD

**Issue by marianne013** [DIRACGrid/DIRAC#6014]
We see this message for each of our CEs in the SiteDirector log. 
```
'Message': "Path /Resources/Computing/ceprod03.grid.hep.ph.ic.ac.uk does not exist or it'\s not a section", '
```
coming from
https://github.com/DIRACGrid/DIRAC/blob/7859236ca8f81c83240fb37af2df60d100992925/src/DIRAC/Resources/Computing/ComputingElement.py#L126-L127

shouldn't that be ceType instead of ceName as the hostname specific options go into the Site ?

**Comment by fstagni** [comment]
I think you might be right. Care to make a fix?

**Comment by marianne013** [comment]
I just tried to fix this, but it turns out ceType is not initialized in this bit of code. So this code can have never done anything (as ceName is also not an option).   
Maybe we should just delete "/Resources/Computing/%s" % self.ceType and remove the loop and just keep /Resources/Computing/CEDefaults ?
That should keep the current behaviour, but get rid of the spurious error.

**Comment by fstagni** [comment]
`self.ceType` is initialized in the actual implementation of CE types, e.g. in ARCComputingElement.py

**Comment by sfayer** [comment]
Hi,

We tried this out but it didn't work as expected... When the CE object is created by the SiteDirector at this point the self.ceType is still uninitialised/set to "" (i.e. an empty string). I think it's probably filled in by setParamaters later in the execution (it does certainly get set correctly at some point).

We played around with this a bit yesterday but didn't understand why this loop was needed: From the meeting earlier it sounds like this bit of code does do something useful when called in the pilot. Perhaps we could do something like the following to make it work for both cases?

```python
sections = ["/Resources/Computing/CEDefaults"]
if self.ceType:
  sections.append("/Resources/Computing/%s" % self.ceType)
for section in sections:
  ...
```

Regards,
Simon
